validator/src/admin_rpc_service.rs

```diff
@@ -266,7 +266,12 @@ impl AdminRpc for AdminRpcImpl {
                 // (rocksdb background processing or some other stuck thread perhaps?).
                 //
                 // If the process is still alive after five seconds, exit harder
-                thread::sleep(Duration::from_secs(5));
+                thread::sleep(Duration::from_secs(
+                    env::var("SOLANA_VALIDATOR_EXIT_TIMEOUT")
```

Comment:
warning: I'm lazy

Reply:
Looks like you set a value of `30` in the CI script; why do we need a larger value ?

Reply:
hey, thanks for reviewing. how about this explanation?: 957e8eb0250

come to think of it, 30 wasn't enough to detect stuck validator termination. so i increased it a bit.

i think this knob will be useful to debug agave-validator's long process exit in the future. fyi, it was exiting successfully when i dig in this issue a bit in the past like 6mo ago...